### ISSUE TITLE
Tables create does not return a complete Table resource

### ISSUE THREAD

**Issue by mziccard** [GoogleCloudPlatform/google-cloud-java#340]
Playing around with table creation I stumbled upon this:

When I successfully create a table I get the following response:
```json
{  
   "kind":"bigquery#table",
   "etag":"\"AxMVOKw7qC1yJvFAglg_iwWnfXE/1uI1j9CjWrEth7qwtgYXqgfPkQo\"",
   "id":"gcloud-devel:gcloud_test_dataset_temp_179c9ccb_4acc_4c8e_b0f3_41d8e9d8ae76.testTable",
   "selfLink":"https://www.googleapis.com/bigquery/v2/projects/gcloud-devel/datasets/gcloud_test_dataset_temp_179c9ccb_4acc_4c8e_b0f3_41d8e9d8ae76/tables/testTable",
   "tableReference":{
      "projectId":"gcloud-devel",
      "datasetId":"gcloud_test_dataset_temp_179c9ccb_4acc_4c8e_b0f3_41d8e9d8ae76",
      "tableId":"testTable"
   },
   "schema":{
      "fields":[
         {  
            "name":"oneField",
            "type":"INTEGER"
         }
      ]
   },
   "type":"TABLE"
}
```
This is not a complete [table resource](https://cloud.google.com/bigquery/docs/reference/v2/tables) as some fields (creationTime, numBytes, numRows, ...) are missing. Do you have any idea why some fields are not set in create's response?

If I get the just created table the response is complete, as expected.

**Comment by aozarov** [comment]
Yes, I tried it as well and I experienced the same issue (`tables.insert` returns an incomplete `Table` object even when I explicitly requested all fields).

This does not seem to be the case for `datasets.insert`.

/cc @jtigani

**Comment by mziccard** [comment]
This was fixed, but now also tables experience issue #466
